test/input/yaml/CustomInput.yaml

```diff
@@ -100,4 +100,7 @@ parity_parameters:
     parity_check_file_index_col: feature_id
     parity_check_file_value_col: streamflow
     parity_check_compare_node: 4186169
+data_assimilation_parameters:
+    wrf_hydro_channel_ID_routelink_file: "../../test/input/geo/routelink/routeLink_subset.nc"
+
```

Comment:
We'll need to build a more bespoke example that includes some USGS timeslice data in the repository. We can follow the pattern we have followed with the Qlaterals and warm states and write out a much more compact csv with just the needed values.